continuousflex/protocols/protocol_genesis.py

```diff
@@ -248,33 +231,33 @@ def _defineParams(self, form):
 
         # Experiments =================================================================================================
         form.addSection(label='EM data')
-        # form.addParam('EMfitChoice', params.EnumParam, label="Cryo-EM Flexible Fitting", default=0,
-        #               choices=['None', 'Volume'], important=True,
-        #               help="Type of cryo-EM data to be processed")
         form.addParam('EMfitChoice', params.EnumParam, label="Cryo-EM Flexible Fitting", default=0,
                       choices=['None', 'Volume (s)', 'Image (s)'], important=True,
                       help="Type of cryo-EM data to be processed")
 
         group = form.addGroup('Fitting parameters', condition="EMfitChoice!=0")
         group.addParam('constantK', params.StringParam, default="10000", label='Force constant (kcal/mol)',
-                      help="Force constant in Eem = k*(1 - c.c.). Note that in the case of REUS, the number of "
+                      help="Force constant in Eem = k*(1 - c.c.). Determines the strengh of the fitting. "
+                           " This parameters must be tuned with caution : "
+                           "to high values will deform the structure and overfit the data, to low values will not "
+                           "move the atom senough to fit properly the data. Note that in the case of REUS, the number of "
                            " force constant value must be equal to the number of replicas, for example for 4 replicas,"
                            " a valid force constant is \"1000 2000 3000 4000\", otherwise you can specify a range of "
                            " values (for example \"1000-4000\") and the force constant values will be linearly distributed "
                            " to each replica."
                       , condition="EMfitChoice!=0")
-        group.addParam('emfit_sigma', params.FloatParam, default=2.0, label="EM Fit Sigma",
+        group.addParam('emfit_sigma', params.FloatParam, default=2.0, label="EM fit gaussian variance",
                       help="Resolution parameter of the simulated map. This is usually set to the half of the resolution"
                         " of the target map. For example, if the target map resolution is 5 Å, emfit_sigma=2.5",
-                      condition="EMfitChoice!=0",expertLevel=params.LEVEL_ADVANCED)
+                      condition="EMfitChoice!=0")
```

Comment:
EMfitChoice!=0 is repeated many times, you may want to consider extracting it as a constant for clarity

Reply:
yes